### DISCUSSION TITLE
Information which can help improve the emulation of various MIDI interfaces

### DISCUSSION THREAD

**Discussion by Ompronce** [86Box/86Box#1721] [General]
**Is your feature request related to a problem? Please describe.**
On the Vogons forum, I saw a thread where a user did tests on a number of sound cards that have UART MPU-401 interfaces on them in order to determine the buffer sizes, speed and more previously unknown information.

**Describe the solution you'd like**
A contributor with more advanced programming experience can make use of this information to improve the MPU-401 emulation of a number of sound cards that are emulated by 86Box.

**Describe alternatives you've considered**
I haven't seen or considered any alternatives as of the time of writing.

**Additional context**
The Vogons forum thread that is mentioned above is linked below:

https://www.vogons.org/viewtopic.php?p=597742

**Comment by OBattler** [comment]
Closing because I moved it to discussions: https://github.com/86Box/86Box/discussions/1719 .
